stackdriver/Rakefile

```diff
@@ -69,14 +69,16 @@ task :console do
 
   require "google/cloud"
   Bundler.require # load all the gems defined in the Gemfile
-  def gcloud; @gcloud ||= Google::Cloud.new; end
+  def gcloud
+    @gcloud ||= Google::Cloud.new
+  end
 
   ARGV.clear
   IRB.start
 end
 
 desc "Generates JSON output from stackdriver .yardoc"
-task :jsondoc => :yard do
+task jsondoc: :yard do
```

Comment:
We just removed the `jsondoc` tasks. Please rebase this PR on master to get the latest.

Reply:
NM, I followed a notification that took me to a version of this PR before the commit of merging to master was added.